File: Algo/hr_implement/2/utopion_tree_height.js
function utopianTree(n) {
    let height =1;   
    if(n>0)   {
        for(let i=1;i<=n;i++){
            if(i%2==0){
                height++;
            } else{
                height = height*2;
            } 
        }
    }    
    return height;
}

console.log(utopianTree(4));

File: Algo/hr_implement/utopion_tree_height.py
"""
Each spring, it doubles in height. Each summer, its height increases by 1 meter.
"""

def utopianTree(n):
	height =  1
	if n > 0:
		for i in range(1,n+1):
			if i%2 == 0:
				height +=1
			else:
				height *=2
	return height

if __name__ == "__main__":
	print utopianTree(0)
	print utopianTree(1)
	print utopianTree(4)